Maestro/subscriptions.json

```diff
@@ -6,6 +6,13 @@
       "vsoProject": "DevDiv",
       "buildDefinitionId": 1084
     },
+    // A build definition capable of running any .ps1 script in the CoreFX repo dev/api branch
+    "corefx-general-dev-api": {
+      "vsoInstance": "devdiv.visualstudio.com",
+      "vsoProject": "DevDiv",
+      "vsoSourceBranch": "dev/api",
```

Comment:
We could change Maestro's code to look in both places, if you think this way is more understandable.

Just changing it here:  https://github.com/dotnet/versions/blob/master/Maestro/src/Microsoft.DotNet.Maestro/Handlers/VsoBuildHandler.cs#L47-L53 to look in both places.